project-koku/koku
# [COST-3667] - Switch to django-tenants

## Jira Ticket

[COST-3667](https://issues.redhat.com/browse/COST-3667)

## Description

This change replaces `django-tenant-schemas` package with `django-tenants` and adds `psycopg2` to pipfile

## Testing

1. Checkout Branch
2. Restart Koku
3. Load test data 
4. You should see ingestion complete successfully
5. You can also hit some endpoints make sure requests are processed fine like;
 - get all sources: http://127.0.0.1:8000/api/cost-management/v1/sources/
 - get all cost models: http://127.0.0.1:8000/api/cost-management/v1/cost-models/

## Notes

...
